tracer/src/Datadog.Trace/Iast/Aspects/System.DirectoryServices/DirectoryEntryAspect.cs

```diff
@@ -10,7 +10,7 @@
 namespace Datadog.Trace.Iast.Aspects.System.DirectoryServices;
 
 /// <summary> DirectoryEntry class aspects </summary>
-[AspectClass("System.DirectoryServices", AspectFilter.StringOptimization)]
+[AspectClass("System.DirectoryServices", new[] { AspectFilter.StringOptimization })]
```

Comment:
nit: missed a chance for .NET 8-isms here 😂 
```suggestion
[AspectClass("System.DirectoryServices", [AspectFilter.StringOptimization])]
```

Reply:
It's never too late :)